DEPS

```diff
@@ -39,10 +39,17 @@ hooks = [
     'action': ['vpython3', 'script/download_rust_deps.py', '--platform', 'ios'],
   },
   {
-    # Download rust deps if necessary for Linux, macOS, Windows
+    # Download rust deps if necessary for Linux
     'name': 'download_rust_deps',
     'pattern': '.',
-    'condition': 'not checkout_android and not checkout_ios',
+    'condition': 'checkout_linux',
+    'action': ['vpython3', 'script/download_rust_deps.py', '--platform', 'linux'],
```

Comment:
I don't understand this change, why do we have to pass the platform for linux when we didn't before?

Reply:
because then we differentiate by arch in the Python script below